src/internal/kopia/conn_test.go

```diff
@@ -33,11 +33,13 @@ func openKopiaRepo(t *testing.T, ctx context.Context) (*conn, error) {
 // unit tests
 // ---------------
 type WrapperUnitSuite struct {
-	suite.Suite
+	tester.Suite
 }
 
 func TestWrapperUnitSuite(t *testing.T) {
-	suite.Run(t, new(WrapperUnitSuite))
+	s := &WrapperUnitSuite{Suite: tester.NewUnitSuite(t)}
+
+	suite.Run(t, s)
```

Comment:
How about:?
```suggestion
	suite.Run(t, &WrapperUnitSuite{Suite: tester.NewUnitSuite(t)})
```

Reply:
oh, that does work. Sometimes golang is weird about passing pointers to freshly created struct instances because they're temporary values